core/templates/dev/head/pages/exploration_editor/editor_tab/StateSolutionEditorDirective.js

```diff
@@ -54,16 +59,10 @@ oppia.directive('stateSolutionEditor', [
 
           ExplorationWarningsService.updateWarnings();
 
-          $scope.isSolutionValid = function() {
-            return ExplorationStatesService.isSolutionValid(
-              EditorStateService.getActiveStateName());
-          };
-
           $scope.correctAnswerEditorHtml = (
             ExplorationHtmlFormatterService.getInteractionHtml(
               stateInteractionIdService.savedMemento,
-              ExplorationStatesService.getInteractionCustomizationArgsMemento(
-                EditorStateService.getActiveStateName()),
+              $scope.getInteractionCustomizationArgsMemento(),
```

Comment:
Bit odd that this differs from the previous arg. Why not stateInteractionCustomizationArgsService.savedMemento?

Reply:
Two fields are returned from ExplorationStatesService for this call, maybe that's why this call was used.

Reply:
Right. But why not replace this function with stateInteractionCustomizationArgsService.savedMemento directly, so that it has parity with the first argument? On looking at getInteractionCustomizationArgsMemento it looks pretty similar to getInteractionIdMemento... so I'm not sure why they're treated differently.

Reply:
Yeah, they are returning the same value, changed it.